src/AnalyzerPowerPack/AnalyzerPowerPack.md

```diff
@@ -0,0 +1,109 @@
+### CA1008: Enums should have zero value. ###
```

Comment:
Absolutely. @michaelcfanning and @genlu are driving this cleanup, which definitely involves making sure all existing analyzers have useful, grammatical documentation. See Michael's comment below.